pages/common/ffmpeg.md

```diff
@@ -25,3 +25,7 @@
 - Remux MKV video to MP4 without re-encoding audio or video streams:
 
 `ffmpeg -i {{input_video}}.mkv -codec copy {{output_video}}.mp4`
+
+- Convert MP4 video to VP9 codec. The CRF value can be from 0–63. Lower values mean better quality. -b:v MUST be 0:
+
+`ffmpeg -i {{input_video}}.mp4 -c:v libvpx-vp9 -crf 30 -b:v 0 {{output_video}}.webm`
```

Comment:
This should probably be rewritten to be similar to the lines above using the long-form arguments (ie. -codec:video).

Also for the sake of surprising the user the least it might be worth adding a audio codec argument, even if it's just a copy, Although webm is pretty picky so it might be worth making the codec opus or something it can handle.